test/com/wsscode/pathom3/connect/runner_test.cljc

```diff
@@ -1021,7 +1021,7 @@
           (graph-response?
             (pci/register
               [(pco/resolver 'users
-                 {::pco/output [{:main-db {:users [:user/id]}}]}
+                 {::pco/output [{:main-db [{:users [:user/id]}]}]}
```

Comment:
This took me a long time to figure out as once I had introduced the change this test started failing and I didn't consider the test to be broken. Once I had a good grasp of the planner code I thought there was a bug in the EQL AST! Eventually I realised the test was wrong... shows I don't have a lot of experience looking at EQL queries else I would have probably picked it up sooner!

Reply:
Good catch, the EQL was valid if the test was about Unions, but seems like not the case here 👍